org.eclipse.lsp4j/src/main/java/org/eclipse/lsp4j/DiagnosticTag.java

```diff
@@ -0,0 +1,48 @@
+/******************************************************************************
+ * Copyright (c) 2019 Mcirosoft.
```

Comment:
I guess the copyright owner should be spelled differently?

Reply:
You've got me! 👍 

Just fixed.